proxy-server/proxy-server.go

```diff
@@ -252,6 +253,7 @@ func handleRequestAndRedirect(res http.ResponseWriter, req *http.Request) {
 		if validateAuthToken(claims) {
 			err = setNewCookie(res, req)
 			if err != nil {
+				log.Printf("Error setting cookie: %+v", err)
```

Comment:
If this actually happens, that was likely a network issue. So the browser will repeat and go through auth loop once more.  Nobody would probably even notice.

Reply:
Oh absolutely. I just didn't want to be in the situation where it's somehow happening and we have no idea what's going on. Having this here costs us nothing.